spring-integration-file/src/test/java/org/springframework/integration/file/remote/session/CachingSessionFactoryTests.java

```diff
@@ -61,6 +62,7 @@ public void testCacheAndReset() {
 		assertTrue(sess1.isOpen());
 		sess1 = cache.getSession();
 		assertEquals("session:1", TestUtils.getPropertyValue(sess1, "targetSession.id"));
+		assertTrue((TestUtils.getPropertyValue(sess1, "targetSession.testCalled", Boolean.class)));
```

Comment:
The `TestSession` is a part of this class, so its `private` properties are visible.
And afterwards we don't need that `@SuppressWarnings("unused")` as well.

Reply:
? 

It's wrapped in a `CachedSession` the `CachedSession.targetSession` is private.